qutip/solver/brmesolve.py

```diff
@@ -198,17 +199,23 @@ def brmesolve(
             a_op = QobjEvo(a_op, args=args, tlist=tlist)
         if isinstance(spectra, str):
             new_a_ops.append(
-                (a_op, coefficient(spectra, args={**args, 'w':0})))
+                (a_op, coefficient(spectra, args={**args, 'w': 0})))
         elif isinstance(spectra, InterCoefficient):
             new_a_ops.append((a_op, SpectraCoefficient(spectra)))
         elif isinstance(spectra, Coefficient):
             new_a_ops.append((a_op, spectra))
+        elif (isinstance(spectra, BosonicEnvironment) or
+              isinstance(spectra, FermionicEnvironment)):
```

Comment:
maybe better not to support fermionic environments here yet (brmesolve needs hermitian coupling operators, and fermionicenvironments have two different power spectra)

Reply:
That's True, I meant that last commit to find out which version of numpy needed the .item() bit, I'll try adapting it to properly do fermionic this weekend, if not I'll keep the BosonicEnvironment only (Possibly also worth accepting bosonic baths as well ?)